app/assets/javascripts/teaspoon/qunit.coffee

```diff
@@ -33,6 +33,7 @@ class Teaspoon.Spec
     return [] unless @spec.failed
     for item in @spec.assertions
       continue if item.result
+      item.message ||= "Expected #{JSON.stringify(item.actual)} to equal #{JSON.stringify(item.expected)}"
```

Comment:
I haven't configured coffeescript/javascript stuff on hound yet, so feel free to ignore.